### DISCUSSION TITLE
Official Raspberry Pi 7" Touchscreen Display

### DISCUSSION THREAD

**Discussion by MrEggHead123** [Geektoolkit/Dynaframe3#178] [Q&A]
Hey absolutely awesome project and the work done so far.

Is there likely to future support for the touchscreen display? i.e. Dynaframe goes back to desktop (or whatever is open) on touch, then after 60seconds relaunches/resumes Dynaframe?

Reason for asking is i use my Pi for HomeAssistant display, and a photo frame when idle. Would be soo cool to use Dynaframe and all its rich features instead of the basic screensaver.

Keep up the awesome work.

Thanks

**Comment by Geektoolkit** [comment]
Thankyou!
I'm hard at work on the successor..Dynaframe pro.  I don't have touchscreen support in either currently, but I can give you a path that may get you close to what you're looking for.   There are web apis documented in the wiki that are just http calls (this is essentially how dynaframe works).  You could potentially call those from home assistant perhaps to control it, though I dont know how you'd exit (can you send an 'escape' on touch or something?.  This may just be work that someone would have to add. I made this open source so the community could jump in and add what they wanted.  I agree it'd be slick to have as a screensaver if the exit portion can be figured out.

**Reply by MrEggHead123** [reply]
Many thanks for your prompt response!

Yeh had a quick look over the API's, sure i could link to "exit" on touch, however is their an API to re-launch?

if you are able to provide a path that would be great, ill have a nosy over the next few days and see if i get anyway and report back!

Thanks again for your awesome work!

**Comment by Geektoolkit** [comment]
oh good point there's no launch api.  it should be in in the home directory, something like home\pi\Dynaframe\ and the executable is just called Dynaframe.  If you can trigger a shell command to launch that you should be golden.  Please dont launch it as sudo though, then it can't find the pictures folder and crashes. something else i had to fix in the next version :)
